docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-prices-feature-integration.md

```diff
@@ -914,19 +912,8 @@ class PriceProductDependencyProvider extends SprykerPriceProductDependencyProvid
             new PriceProductOfferPriceProductDimensionExpanderStrategyPlugin(),
         ];
     }
-    
-    /**
-     * @return \Spryker\Zed\PriceProductExtension\Dependency\Plugin\PriceProductValidatorPluginInterface[]
-     */
-    protected function getPriceProductValidatorPlugins(): array
-    {
-        return [
-            new PriceProductVolumeValidatorPlugin(),
-        ];
-    }
```

Comment:
lets leave it in case it is not presented in original feature IG

Reply:
git it. reverted